tests/reference/asr-constants1-5828e8a.stderr

```diff
@@ -0,0 +1,15 @@
+Traceback (most recent call last):
+  File "$DIR/src/bin/../runtime/lpython_parser.py", line 10, in <module>
+    a = ast.parse(input, type_comments=True)
+  File "/home/abdelrahmankh/anaconda3/envs/lp/lib/python3.10/ast.py", line 50, in parse
+    return compile(source, filename, mode, flags,
+  File "<unknown>", line 511
+    res:c32 a
+            ^
+SyntaxError: invalid syntax
+The command 'python $DIR/src/bin/../runtime/lpython_parser.py $DIR/src/bin/../runtime/lpython_builtin.py' failed.
+semantic error: The file '$DIR/src/bin/../runtime/lpython_builtin.py' failed to parse
+ --> tests/constants1.py:3:9
+  |
+3 |     b = bin(5)
+  |         ^^^^^^ 
```

Comment:
This error message I think does not seem like the right reference test. I think these tests should not error out, should they?

Reply:
yes its a mistake ,i didn't update tests after i fixed some errors that make all referance tests get errors.